Source/Extensions/Blazorise.Charts.DataLabels/Blazorise.Charts.DataLabels.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <Import Project="..\..\..\Build\Blazorise.props" />
+
+  <PropertyGroup>
+    <PackageTags>blazorise blazor components chart datalabels</PackageTags>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>1591</NoWarn>
+  </PropertyGroup>
```

Comment:
Isnt this already common to all? Or should?

Reply:
Not for extensions. `<NoWarn>1591</NoWarn>` is used so that we don't get too many warnings. But, yeah, in the long run we will need to comment all extensions and remove NoWarn.